src/search/landmarks/landmark_factory_reasonable_orders_hps.h

```diff
@@ -0,0 +1,30 @@
+#ifndef LANDMARKS_LANDMARK_FACTORY_REASONABLE_ORDERS_HPS_H
+#define LANDMARKS_LANDMARK_FACTORY_REASONABLE_ORDERS_HPS_H
+
+#include "landmark_factory.h"
+
+namespace landmarks {
+class LandmarkFactoryReasonableOrdersHPS : public LandmarkFactory {
```

Comment:
I assume that all of the implementation in the cc-file is mostly moved from the previous factory class, isn't it? Given that I'm lazy I would prefer to not look at the cc-file in all details.

Reply:
Yes, the implementation is copy pasted. I don't want to exclude that I might have done some minor cosmetic changes, but I doubt it.